client/src/app/gateways/export/html-to-pdf.service/html-to-pdf.service.ts

```diff
@@ -482,11 +482,11 @@ export class HtmlToPdfService {
                         case `text-decoration`: {
                             switch (value) {
                                 case `underline`: {
-                                    styleObject.decoration = value;
+                                    styleObject.decoration = (styleObject.decoration ?? []).concat(value);
```

Comment:
As this is currently not officially supported by pdfmake I opened PRs in pdfmake doc and DefinetlyTyped.